bundles/org.openhab.binding.unifi/src/main/java/org/openhab/binding/unifi/internal/handler/UniFiPoePortHandler.java

```diff
@@ -0,0 +1,166 @@
+                .getConfiguration().get(CHANNEL_ENABLE_PARAMETER_MODE);
+        poeEnableMode = channelConfigPoeEnableMode.isBlank() ? CHANNEL_ENABLE_PARAMETER_MODE_AUTO
+                : channelConfigPoeEnableMode;
+        return true;
+    }
+
+    @Override
+    protected @Nullable Map<Integer, UniFiPortTable> getEntity(final UniFiController controller) {
+        return controller.getSwitchPorts(config.getMacAddress());
+    }
+
+    @Override
+    protected void refreshChannel(final Map<Integer, UniFiPortTable> ports, final ChannelUID channelUID) {
+        final UniFiPortTable port = getPort(ports);
+        if (port == null) {
+            return;
+        }
+        final String channelID = channelUID.getIdWithoutGroup();
+        final State state;
+
+        switch (channelID) {
+            case CHANNEL_ONLINE:
+                state = OnOffType.from(port.isUp());
+                break;
+            case CHANNEL_PORT_PEO_ENABLE:
+                state = OnOffType.from(port.isPoeEnable());
+                break;
+            case CHANNEL_PORT_PEO_MODE:
+                state = StringType.valueOf(port.getPoeMode());
+                break;
+            case CHANNEL_PORT_PEO_POWER:
+                state = new QuantityType<Power>(Double.valueOf(port.getPoePower()), Units.WATT);
+                break;
+            case CHANNEL_PORT_PEO_VOLTAGE:
+                state = new QuantityType<ElectricPotential>(Double.valueOf(port.getPoeVoltage()), Units.VOLT);
+                break;
+            case CHANNEL_PORT_PEO_CURRENT:
+                state = new QuantityType<ElectricCurrent>(Double.valueOf(port.getPoeCurrent()), MILLI(Units.AMPERE));
+                break;
+            default:
+                state = UnDefType.UNDEF;
+        }
+        updateState(channelUID, state);
+    }
+
+    private @Nullable UniFiPortTable getPort(final Map<Integer, UniFiPortTable> ports) {
+        return ports.get(config.getPortIdx());
+    }
+
+    @Override
+    protected boolean handleCommand(final UniFiController controller, final Map<Integer, UniFiPortTable> ports,
+            final ChannelUID channelUID, final Command command) throws UniFiException {
+        final String channelID = channelUID.getIdWithoutGroup();
+
+        switch (channelID) {
+            case CHANNEL_PORT_PEO_ENABLE:
+                if (command instanceof OnOffType) {
+                    handleModeCommand(controller, ports, getPort(ports),
+                            OnOffType.ON == command ? poeEnableMode : CHANNEL_ENABLE_PARAMETER_MODE_OFF);
+                }
+                break;
+            case CHANNEL_PORT_PEO_MODE:
+                if (command instanceof StringType) {
+                    handleModeCommand(controller, ports, getPort(ports), command.toFullString());
+                }
+                break;
+            default:
+                logger.warn("Ignoring unsupported command = {} for channel = {} - valid commands types are: OnOffType",
```

Comment:
Consistency: In UniFiClientThingHandler logging of unsupported command was reduced to `logger.info`.